REQUIRE

```diff
@@ -5,4 +5,4 @@ ForwardDiff 0.7.0
 LineSearches 5.0.0
 DiffBase
 NLSolversBase 6.0.0
-Reexport
+Reexport
```

Comment:
Why remove the newline char?

Reply:
This is something my editor does; I'm not sure why.

Reply:
Sublime?

Reply:
VS Code

Reply:
Added a newline here, I think.